docs/source-pytorch/data/infinite.rst

```diff
@@ -0,0 +1,49 @@
+:orphan:
```

Comment:
iterable-style does not imply infinite. Maybe consider renaming it to iterable.rst? In case you decide to rename, don't forget to update  `:button_link: infinite.html` above :)